build_modules/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ repository: https://github.com/dart-lang/build/tree/master/build_modules
 resolution: workspace
 
 environment:
-  sdk: '>=3.5.0 <3.7.0-z'
+  sdk: '>=3.7.0-82.0.dev <3.8.0-z'
```

Comment:
We can only depend on released dev versions so we might need to wait until the next release (3.7.0-81.0.dev is the current most recent one)